cost/aws/s3_bucket_size/aws_bucket_size.pt

```diff
@@ -229,9 +248,12 @@ end
 
 policy "aws_oversized_buckets" do
   validate $aws_oversized_buckets do
-    summary_template "{{ rs_project_name }} (Account ID: {{ rs_project_id }}): {{ len data }} Oversized Buckets Found in AWS"
+    summary_template "{{ len data }} Oversized Buckets Found in AWS Account ID: {{with index data 0}}{{ .accountId }}{{end}}"
```

Comment:
Thinking we need to update this summary to reflect the same pattern as the others.

Reply:
This reads correctly. The other way is horrible for this policy.